src/main/java/mate/academy/springbootwebgreqit/controller/ShoppingCartController.java

```diff
@@ -21,25 +22,25 @@ public class ShoppingCartController {
     private final ShoppingCartService shoppingCartService;
 
     @GetMapping
-    public ShoppingCartDto getShoppingCartForCurrentUser(@RequestParam Long userId) {
-        return shoppingCartService.getShoppingCartForCurrentUser(userId);
+    public ShoppingCartDto getShoppingCartForCurrentUser(Authentication authentication) {
+        return shoppingCartService.getShoppingCartForCurrentUser(authentication);
```

Comment:
It's better to create separate method that return Long userId from Authentication